DQMOffline/Trigger/plugins/JetMETMonitor.cc

```diff
@@ -0,0 +1,1056 @@
+  desc.add<edm::InputTag>( "electrons",edm::InputTag("gedGsfElectrons") );
+  desc.add<edm::InputTag>( "muons",    edm::InputTag("muons") );
+  desc.add<int>("njets",      0);
+  desc.add<int>("nelectrons", 0);
+  desc.add<int>("nmuons",     0);
+  desc.add<double>("ptcut",   0);
+  desc.add<std::string>("metSelection", "pt > 0");
+  desc.add<std::string>("jetSelection", "pt > 20");
+  desc.add<std::string>("calojetSelection", "pt > 20");
+  desc.add<std::string>("eleSelection", "pt > 0");
+  desc.add<std::string>("muoSelection", "pt > 0");
+  desc.add<bool>("ispfjettrg",    true);
+  desc.add<bool>("iscalojettrg",  false);
+  desc.add<bool>("ismettrg",      false);
+  desc.add<bool>("isjetFrac", false);
+
+  edm::ParameterSetDescription genericTriggerEventPSet;
+  genericTriggerEventPSet.add<bool>("andOr");
+  genericTriggerEventPSet.add<edm::InputTag>("dcsInputTag", edm::InputTag("scalersRawToDigi") );
+  genericTriggerEventPSet.add<std::vector<int> >("dcsPartitions",{});
+  genericTriggerEventPSet.add<bool>("andOrDcs", false);
+  genericTriggerEventPSet.add<bool>("errorReplyDcs", true);
+  genericTriggerEventPSet.add<std::string>("dbLabel","");
+  genericTriggerEventPSet.add<bool>("andOrHlt", true);
+  genericTriggerEventPSet.add<edm::InputTag>("hltInputTag", edm::InputTag("TriggerResults::HLT") );
+  genericTriggerEventPSet.add<std::vector<std::string> >("hltPaths",{});
+//  genericTriggerEventPSet.add<std::string>("hltDBKey","");
+  genericTriggerEventPSet.add<bool>("errorReplyHlt",false);
+  genericTriggerEventPSet.add<unsigned int>("verbosityLevel",1);
+
+  desc.add<edm::ParameterSetDescription>("numGenericTriggerEventPSet", genericTriggerEventPSet);
+  desc.add<edm::ParameterSetDescription>("denGenericTriggerEventPSet", genericTriggerEventPSet);
+
+  edm::ParameterSetDescription histoPSet;
+  edm::ParameterSetDescription metPSet;
+  fillHistoPSetDescription(metPSet);
+  histoPSet.add<edm::ParameterSetDescription>("metPSet", metPSet);  
+  std::vector<double> bins = {0.,20.,40.,60.,80.,90.,100.,110.,120.,130.,140.,150.,160.,170.,180.,190.,200.,220.,240.,260.,280.,300.,350.,400.,450.,1000.}; // Jet pT Binning
+  histoPSet.add<std::vector<double> >("jetptBinning", bins);
+
+  edm::ParameterSetDescription lsPSet;
+  fillHistoLSPSetDescription(lsPSet);
+  histoPSet.add<edm::ParameterSetDescription>("lsPSet", lsPSet);
+
+  desc.add<edm::ParameterSetDescription>("histoPSet",histoPSet);
+
+  descriptions.add("jetmetMonitoring", desc);
+}
+
+bool JetMETMonitor::isBarrel(double eta){
+  bool output = false;
+  if (fabs(eta)<=1.3) output=true;
+  return output;
+}
+
+//------------------------------------------------------------------------//
+bool JetMETMonitor::isEndCapM(double eta){
+  bool output = false;
+  if (fabs(eta) <= 3.0 && fabs(eta) > 1.3 && (eta/fabs(eta) < 0) ) output=true;
+  return output;
+}
+/// For Hcal Endcap Plus Area
+bool JetMETMonitor::isEndCapP(double eta){
```

Comment:
I didn't realize we have a map for detector acceptance.  So I hardcoded ...
Do you have a instruction or twiki to explain  how to use this map?